.gitignore

```
@@ -392,6 +392,5 @@ docker/
 
 # Ignore files managed by gitbook fetch step
 apps/nextjs-website/.tmp/
```

I think docs folder shouldn't be pushed to repository and is correct to be ignored or there is something i have missed ? :thinking: 
```suggestion
apps/nextjs-website/.tmp/
apps/nextjs-website/docs/
```